HACKING.md

```diff
@@ -234,13 +234,21 @@ To get started from a pristine tree you want to do this:
 ./mkversion.sh
 cd cmd/
 autoreconf -i -f
-./configure --prefix=/usr --libexecdir=/usr/lib/snapd
+./configure --prefix=/usr --libexecdir=/usr/lib/snapd --enable-nvidia-multiarch --with-host-arch-triplet=$(dpkg-architecture -qDEB_HOST_MULTIARCH)"
 ```
 
 This will drop makefiles and let you build stuff. You may find the `make hack`
 target, available in `cmd/snap-confine` handy, it installs the locally built
 version on your system and reloads the apparmor profile.
 
+Note that this assumes you are on Ubuntu, with the `--with-host-arch-triplet` 
+configure option, that as well as the `--enable-nvidia-multiarch` option enable
+sharing of the host's graphics drivers and libraries with snaps and is generally
+necessary to run/test graphical applications with your local version of 
+snap-confine. There is also the `--enable-nvidia-biarch` option which can be 
+used if you are on other distros, but if this is the case there are likely other
+additional options you will need to use anyways.
```

Comment:
Looks good! Here's my suggestion for trying to simplify the long sentence (feel free to ignore/use what you think is best): 

```markdown
Note, the above configure options assume you are on Ubuntu and are generally
necessary to run/test graphical applications with your local version of
snap-confine. The `--with-host-arch-triplet` option sets your specific 
architecture and `--enable-nvidia-multiarch` allows the host's graphics drivers
and libraries to be shared with snaps. If you are on a distro other than
Ubuntu, try `--enable-nvidia-biarch` (though you'll likely need to add further
system-specific options too).
```

Reply:
Thanks, applied!